content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -247,9 +249,25 @@ message ContainerDevices {
 }
 ```
 
+### `GetAllocatableResources` gRPC endpoint {#grpc-endpoint-getallocatableresources}
+
+{{< feature-state state="beta" for_k8s_version="v1.23" >}}
+
 GetAllocatableResources provides information on resources initially available on the worker node.
 It provides more information than kubelet exports to APIServer.
 
+{{< note >}}
+`GetAllocatableResources` should only be used to evaluate [allocatable](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)
+resources on a node. If the goal is to evaluate free/unallocated resources it should be used in
+conjunction with the List() endpoint. The result obtained by `GetAllocatableResources` would remain
+the same unless the underlying resources exposed to kubelet change. This happens rarely but when
+it does (for example: hotplug/hotunplug, device health changes), client is expected to call
+`GetAlloctableResources` endpoint.
+However, calling GetAllocatableResources` endpoint is not sufficient in case of cpu and/or memory
```

Comment:
it seems a typo sneaked in. Note lack of -> ` <- preceding `GetAllocatableResources`

Reply:
Ah, good catch! Fixing now

Reply:
Fixed!